jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -166,9 +262,9 @@ YUI.add('user-profile', function() {
             <div className="inner-wrapper">
               <juju.components.UserProfileHeader
                 avatar=""
-                bundleCount={0}
-                charmCount={0}
-                environmentCount={this.state.envList.length}
+                bundleCount={bundleCount || 0}
```

Comment:
I'd prefer this logic to be in the definition - I don't feel too strongly though :)